src/routes/Tool/Interests.tsx

```diff
@@ -49,15 +49,16 @@ const Interests = () => {
           <h1 className="mb-5 mt-7 text-heading-1 text-black">{t('tool.interests.heading-1')}</h1>
 
           <div className="pb-7">
-            <p className="mb-6 text-body-md font-arial text-black">{t('tool.interests.page-description')}</p>
-            <p className="text-body-md font-arial text-black">{t('tool.interests.field-description')}</p>
+            <p className="mb-6 text-body-md text-black">{t('tool.interests.page-description')}</p>
+            <p className="text-body-md text-black">{t('tool.interests.field-description')}</p>
```

Comment:
Text styles could be given just once for the parent `div` for these `p`-tags.

And to also get rid of the margin for first `p` tag, it could be changed like this:

```tsx
<div className="pb-7 text-body-md text-black gap-6 flex flex-col">
  <p>{t('tool.interests.page-description')}</p>
  <p>{t('tool.interests.field-description')}</p>
</div>
```

Reply:
Nice! I did this change here and also on competences page so they match.